### PR TITLE
Pushes New Rules into the Server HTML Rules Button

### PR DESCRIPTION
**Pull request by GhostEcho**
Does as the title says

### CODE REVIEW

File: html/rules.html [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
-<p>4. The role you spawn in as is the role you should play, joining as a role to desert or otherwise not play it, gets you a jobban.</p>
-<p>5. In-game administration rulings are final. If you have a problem with the ruling, take it to another member of staff. Fishing for admins is not recommended.</p>
-<p>6. Faction Metagaming. Do not move to the enemy factions base until at least 40 minutes into the round. Claiming to have &lsquo;found it&rsquo; is a bad excuse, and isn&rsquo;t going to work.</p>
-<p>7. Bad Deathclaw has a focus towards roleplay. Roleplay a believable character that could exist in the Fallout universe. Shitty memes and low roleplay gimmicks will not end well.</p>
-<p>8. Do not be unnecessarily hostile. Arguments are fine, being an asshole for no reason isn't. This includes taking it to OOC chat and making trouble.</p>
-<p>9. Do not attack or harm AFK/SSD players. You are free to take their items, but not attack them.</p>
-<p>10. No rule lawyering.</p>
+<p>0. Don’t be a Dick. If you are intentionally ruining another players experience or going out of your way to negatively impact the round, you can be punished for it..</p>
```
Comment (CameronWoof) [inline html/rules.html]: Decapitalize "Dick". Remove one of the two periods.

File: html/rules.html [hunk 2]
```diff
@@ -1,75 +1,75 @@
...
+<p>    
+	<br /><strong>General Rules</strong></p>
+<p>1. Admin Rulings are final. If you wish to contest a ruling, please go to the discord. In each situation, an admins ruling is how the rules are to be treated in that given situation.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Decapitalize Rulings. "... an admin**'**s ruling

File: html/rules.html [hunk 3]
```diff
@@ -1,75 +1,75 @@
...
+<p>2. Don’t use OOC information IC. Don’t leak IC information to OOC.</p>
+<p>3. Play as the role you spawn in as, joining as a role to desert or otherwise undermine your faction is bannable.</p>
+<p>4. Bad Deathclaw is an RP server, and roleplay is expected. Failing to do so is punishable..</p>
```
Comment (CameronWoof) [inline html/rules.html]: One period.

File: html/rules.html [hunk 4]
```diff
@@ -1,75 +1,75 @@
...
+<p>4. Do not attack, assault or harm SSD/AFK players. You may take their equipment, but dragging them off or murdering them is not okay. Catatonic Players are free game.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Decapitalize Players after Catatonic.

File: html/rules.html [hunk 5]
```diff
@@ -1,75 +1,75 @@
...
+<p>5. No Rules Lawyering.</p>
```
Comment (CameronWoof) [inline html/rules.html]: This is both covered by the "admin's ruling is final" clause and doesn't explain what it actually means. Suggest removal.

File: html/rules.html [hunk 6]
```diff
@@ -1,75 +1,75 @@
...
+<p>6. Name your character appropriately. Inappropriate names may result in a ban. Raider names may be ridiculous, but should not reference OOC memes or pop culture..</p>
```
Comment (CameronWoof) [inline html/rules.html]: One period.

File: html/rules.html [hunk 7]
```diff
@@ -1,75 +1,75 @@
...
 <p>
-    <br /><strong>B. Approach, Motivation and Escalation</strong></p>
-<p>1. Do not attack someone wordlessly.</p>
-<p>2. Establish hostility before attacking. This means using your words. Wait for them to react, whether words, brandishing a weapon, violence, or compliance.</p>
-<p>3. Escalation comes with reasoning and is mostly limited by role background. You shouldn&rsquo;t see Wastelanders and Settlers being mass-murderers, but Raiders are more likely.</p>
-<p>4. Fleeing confrontation constitutes escalation.</p>
-<p>5. If you suspect the other player is going to fail to escalate properly, it&rsquo;s better to let it pan out, then shoot first and have -you- look like the villain.</p>
+    <br /><strong>Escalation Rules</strong></p>
+<p>1. Legion and NCR do not need to escalate on one another, in any way. They may attack each other for any reason at any time.</p>
+<p>2. No other parties may enter into combat with Legion or NCR without being in recognisable faction uniform, of either Legion or NCR. Armor and Helmet count appropriately.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Decapitalize "helmet." Alter wording to "A recognisable faction uniform must include their armor and helmet."

File: html/rules.html [hunk 8]
```diff
@@ -1,75 +1,75 @@
...
+<p>3. Legion and NCR must still escalate on any other party, in every situation.</p>
+<p>4. Escalation requires a motive, as well as verbal interaction to establish hostile intent.</p>
+<p></p>
+<p>Escalation requires a reaction from the party being escalated on. This reaction can be drawing their own weapon, running away, popping pre-combat meds and so forth.</p>
+<p>If someone is attempting to negotiate or is complying with your demands, you should not assault them for trying to negotiate, but instead interact with them. If their case is bad, you may still kill them.</p>
+<p>If someone has used the Surrender Emote, you should not harm them unless they subsequently attempt to flee or attack you.</p>
```
Comment (CameronWoof) [inline html/rules.html]: decapitalize Emote

File: html/rules.html [hunk 9]
```diff
@@ -1,75 +1,75 @@
...
+<p>If you are caught in the crossfire of a fight, that is not an excuse to join either sides conflict. You should get out of the way. </p>
+<p>You may still enter into conflict with non-faction groups, but should escalate clearly and openly, and wait at least 5 seconds before attempting any hostile actions. We can tell if you don’t.</p>
+<p>Escalation requires a reaction from the party being escalated on. This reaction can be drawing their own weapon, running away, popping pre-combat meds and so forth.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Duplicate sentence

File: html/rules.html [hunk 10]
```diff
@@ -1,75 +1,75 @@
...
+<p></p>
+<p>If someone is attempting to negotiate or is complying with your demands, you should not assault them for trying to negotiate, but instead interact with them. If their case is bad, you may still kill them.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Ditto

File: html/rules.html [hunk 11]
```diff
@@ -1,75 +1,75 @@
...
+<p>If someone has used the Surrender Emote, you should not harm them unless they subsequently attempt to flee or attack you.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Ditto

File: html/rules.html [hunk 12]
```diff
@@ -1,75 +1,75 @@
...
+<p>If you are caught in the crossfire of a fight, that is not an excuse to join in either sides conflict. You should get out of the way.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Ditto

File: html/rules.html [hunk 13]
```diff
@@ -1,75 +1,75 @@
...
 <p>
-    <br /><strong>D. Alignment and Siding</strong></p>
-<p>1.Wastelanders and raiders are allowed to join and side with any of the factions.</p>
-<p>2. No hostile action is allowed to be taken while siding with a faction unless you are in their full uniform and attire. Unless you express your verbal allegiance during escalation, you will not be considered valid.</p>
-<p>3. When you have joined a faction, you should wear as much of their uniform as is available, or verbally side with them, when situations are escalating.</p>
-<p>4. All NCR/BOS/Legion Roles, are beholden to IC Rules, and in the case of their Leaders/Higher Ranks, OOC Rules.</p>
-<p>5. Repeatedly breaking IC rules can result in admin intervention.</p>
+    <br /><strong>Uniform and Siding</strong></p>
+<p>1. Faction uniforms are to be kept at all times unless deserting or infiltrating. (Expect consequences if you desert your respective faction!)</p>
+<p>2. To participate in a faction raid/conflict, you MUST be in that factions uniform, or be clearly identifiable through the use of the Faction Tabard item.</p>
```
Comment (CameronWoof) [inline html/rules.html]: This item does not exist. Refer to the armor and helmet requirement further up.

File: html/rules.html [hunk 14]
```diff
@@ -1,75 +1,75 @@
...
-    <br /><strong>E. Uniforms and Armor</strong></p>
-<p>1. Faction uniforms are to be kept at all times unless deserting or infiltrating non-lethally.
-    <br />(Expect consequences if you desert your respective faction!)</p>
-<p>2. Faction armor should only be changed out of for the purposes of infiltration.</p>
-<p>3. Faction uniforms can be used by anyone, but fall under these rules, and normal escalation rules. No surprise murders, no matter how much you like.</p>
-<p>4. If dressed as a member of a faction, you are treated as one for matters of escalation and conflict.</p>
-<p>5. When infiltrating, it is important to let staff know when you do so. It allows them to be aware of it before issues occur.</p>
+    <br /><strong>Factions!</strong></p>
+<p>1. Faction Rules, though Primarily IC, may be punished OOCly.</p>
+<p>2. They are primarily enforced through IC punishment by the factions higher ranks.</p>
```
Comment (CameronWoof) [inline html/rules.html]: faction**'**s

File: html/rules.html [hunk 15]
```diff
@@ -1,75 +1,75 @@
...
-<p>4. Legion ranks are defined by equipment. Survivors traditionally reform the hierarchy by taking the armor of their dead superiors, to assume their position.</p>
-<p>5. Caesar&rsquo;s Legion consider women inferior, this delegates them to support and slavery, rarely if ever do they accept a woman as worthy as more than a slave.</p>
-<p>6. NCR are typically strung up on a cross when captured (and if it is deem fitting, they can be turned into slaves),as are chem users.</p>
-<p>7. Men are to be trained as Legionaries, while women are to be trained as healers and farmers, as well as cooks.</p>
-<p>8. Caesar's Legion is comprised of mainly tribals, meaning most of them do not know how to read, however the Centurions of war camps are typically somewhat educated.</p>
-<p>9. Camp Followers are entirely loyal to the Legion. Playing them as anything but is grounds for a jobban. There are reasonable exceptions to this, such as overt abuse by the Legion, who should at least treat you as an asset.</p>
+<p>1. All combat personnel must have a Roman/Latin styled name and must be male.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Inconsistent spacing

File: html/rules.html [hunk 16]
```diff
@@ -81,74 +81,43 @@
...
-<p>3. Never spawn the same character twice.
-    <br />If you die, do not attempt to resume your prior life.
-    <br />If you happen to spawn with a used name, contact admins immediately.</p>
-<p>4. When cloned, the time you were scanned is the point you remember from. You will not recall who killed you, or even current information.
-    <br />When cloned via pod-people means, you are functionally a new person, having only the personality and traits of the person recreated, without their memory.
-    <br />Roleplay Appropriately!</p>
-<p>5. Do not kill braindeads or AFK&rsquo;s. Looting is fair game, but don't kill them. When they are a member of your faction, shove them into a secure place at your base.</p>
+<p>2. The Brotherhood have an established Chain of Command, known as the Chain That Binds. This system prevents Paladins from ordering around initiates assigned to a Scribe or Knight, unless no Scribes or Knights are responsive.</p>
+<p>3. The Paladin is responsible for all operations relating to the Security of the Bunker, as well as any personnel entering or leaving.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Decapitalize Security

File: html/rules.html [hunk 17]
```diff
@@ -81,74 +81,43 @@
...
+<p>4. The Head Scribe is responsible for the internal operations of the Bunker, relating to research, personnel management, and interior design, where security is not a concern.</p>
+<p>5. When the chain that binds cannot be followed due to a lack of personnel, the Paladin retains general command.</p>
```
Comment (CameronWoof) [inline html/rules.html]: Capitalize Chain That Binds

File: html/rules.html (hunk 13 above)
Comment (GhostEcho) [reply html/rules.html]: It will exist soon

Comment (CameronWoof) [reply html/rules.html]: Then add this line when it does exist. Jumping the gun is why we had rules referring to bitter drink.

**Pull request merged**